0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -22,81 +22,42 @@ SPSP messages MUST be exchanged over HTTPS.
 
 ### Operation
 
-Any SPSP recipient will create a receiving HTTP endpoint called the *receiver*. The sender can query this endpoint to get information about the type of payment that can be made to this receiver. The sender also uses the receiver endpoint to set up the ILP payment.
+Any SPSP recipient will create a receiving HTTP endpoint called the *receiver*. The sender can query this endpoint to get information about the type of payment that can be made to this receiver. The sender can set up and send multiple ILP payments using the details provided by the receiver.
 
 ### Definitions
 
-#### SPSP Client
-
-The sender application that uses SPSP to interact with the SPSP Server.
-
-#### SPSP Server
-
-The server used on the recipient's side to handle SPSP requests.
-
-#### Receiver
-
-The specific HTTP endpoint on the SPSP used for setting up a payment.
+* **SPSP Client** - The sender application that uses SPSP to interact with the SPSP Server
+* **SPSP Server** - The server used on the recipient's side to handle SPSP requests
+* **Receiver** - The specific HTTPS endpoint on the SPSP server used for setting up a payment
+* **ILP Module** - Software included in the SPSP Client and Server that sends and receives Interledger payments
```

Comment:
This is confusing because the `ilp` module includes an SPSP client. Why not refer to it as a ledger plugin?